src/OperatorEnumConstruction.jl

```diff
@@ -291,14 +343,20 @@ function _extend_operators(operators, skip_user_operators, kws, __module__::Modu
             end
         end
         for (op, func) in enumerate($(operators).unaops)
-            local $f = Symbol(func)
+            local $f_outside =
+                typeof(func) <: Broadcast.BroadcastFunction ? Symbol(func.f) : Symbol(func)
```

Comment:
```suggestion
                typeof(func) <: $(Broadcast).BroadcastFunction ? Symbol(func.f) : Symbol(func)
```
just in case a user has another symbol in their namespace for `Broadcast`, it's good to pass the exact module object within the quoted

(here and elsewhere below)